main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"foxygo.at/s/httpe"
+)
+
+var (
+	// Semver holds the version exposed at /version and passed via linker flag on CI.
+	Semver = "undefined"
+	// CommitSha holds the commit exposed at /version and passed via linker flag on CI.
+	CommitSha = "undefined"
+)
+
+func main() {
+	fmt.Println("Listening on http://localhost:8080")
```

Comment:
i know you put this url here so you can click on it on the output, but you're listening on all interfaces not localhost. if you listen on localhost, you cannot connect to it from outside. I dont like misleading log messages - they trip you up.

How about :`Listening on :8080 (accessible on http://localhost:8080)` or something?

Reply:
ok, done.